### PR TITLE
[elixir/it] elixir

### PR DESCRIPTION
**Pull request by kino90**

### CODE REVIEW

**Comment by vendethiel** [comment]
Thanks! :+1:

File: it-it/elixir-it.html.markdown [hunk 1]
```diff
@@ -0,0 +1,417 @@
+---
+language: elixir
+contributors:
+    - ["Joao Marques", "http://github.com/mrshankly"]
+    - ["Dzianis Dashkevich", "https://github.com/dskecse"]
+    - ["Luca Maroni", "https://github.com/kino90"]
+filename: learnelixir.ex
```
Comment (vendethiel) [inline it-it/elixir-it.html.markdown]: please add `lang: it-it`

File: it-it/elixir-it.html.markdown [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+# Atomi, che sono literals, una costante con un nome. Iniziano con `:`.
+:ciao # atomo (Atom)
+
+# Tuple che sono salvati in celle di memoria contigue.
+{1,2,3} # tupla (Tuple)
+
+# Possiamo accedere ad un elemento di una tupla con la funzione `elem`:
+elem({1, 2, 3}, 0) #=> 1
+
+# Liste, che sono implementate come liste come linked lists.
```
Comment (geoffliu) [inline it-it/elixir-it.html.markdown]: Typo

**Comment by kino90** [comment]
Thanks for the feedback! :+1:

**Pull request closed without merging**